android/src/main/java/com/adyenreactnativesdk/AdyenCheckout.kt

```diff
@@ -0,0 +1,136 @@
+package com.adyenreactnativesdk
+
+import android.content.Context
+import android.content.Intent
+import androidx.activity.result.ActivityResultCaller
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContract
+import com.adyen.checkout.components.base.IntentHandlingComponent
+import com.adyen.checkout.dropin.DropIn
+import com.adyen.checkout.dropin.DropInCallback
+import com.adyen.checkout.dropin.DropInResult
+import com.adyenreactnativesdk.action.ActionHandler
+import com.adyenreactnativesdk.component.dropin.ReactDropInCallback
+import com.adyenreactnativesdk.component.googlepay.AdyenGooglePayComponent
+import java.lang.ref.WeakReference
+
+/**
+ * Umbrella class for setting DropIn and Component specific parameters
+ */
+class AdyenCheckout {
```

Comment:
Since this class only has a `companion object` it is nicer to make it `object AdyenCheckout`